eng/Versions.props

```diff
@@ -37,7 +37,7 @@
     manually enabled by updating the metadata.
   -->
   <ItemGroup>
-    <ProjectServicingConfiguration Include="Microsoft.WindowsDesktop.App.Ref" PatchVersion="0" />
+    <ProjectServicingConfiguration Include="Microsoft.WindowsDesktop.App.Ref" PatchVersion="$(PatchVersion)" />
```

Comment:
Should we only be servicing ref packs if they contain a fix? So for instance hardcode this to 1 which is the current PatchVersion, and then in the next servicing when PatchVersion becomes 2, if we don't need to service the ref pack, it will stay with `1` value here and then it will not produce any ref packs.

Reply:
I'll change it.